testsuite/standalone/src/test/java/org/jboss/as/test/integration/logging/profiles/Log4j2LoggingProfilesTestCase.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ *
+ * Copyright 2020 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.as.test.integration.logging.profiles;
+
+import java.io.FilePermission;
+import java.security.Permission;
+import java.util.PropertyPermission;
+
+import org.jboss.as.test.integration.logging.Log4j2ServiceActivator;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.runner.RunWith;
+import org.wildfly.core.testrunner.ServerSetup;
+import org.wildfly.core.testrunner.WildflyTestRunner;
+
+/**
+ * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
+ */
+@RunWith(WildflyTestRunner.class)
+@ServerSetup(AbstractLoggingProfilesTestCase.LoggingProfilesTestCaseSetup.class)
+public class Log4j2LoggingProfilesTestCase extends AbstractLoggingProfilesTestCase {
+
+    public Log4j2LoggingProfilesTestCase() {
+        super(Log4j2ServiceActivator.class, 2);
+    }
+
+    @Override
+    protected void processDeployment(final JavaArchive deployment) {
+        final Permission[] permissions = {
```

Comment:
@jamezp , @boris-unckel why are these permissions needed here? I don't see them in akin tests like `Log4jLoggingProfilesTestCase` .... is the customer required to grant these permissions to the deployment?

Reply:
@tommaso-borgato I tried to explain the best I could in the comments. These are all because of the way the log4j2 API loads things so we have no control over it. I've made a note to update the future documentation for this. This documentation change will not be part of this RFE because it's going to be a large change and I don't want to hold this RFE up based a large documentation re-do.

Reply:
I just created https://issues.redhat.com/browse/WFLY-14160 to track the documentation changes as a whole.